app/Services/Apis/MarketingAPI.php

```diff
@@ -0,0 +1,71 @@
+<?php namespace services\apis;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use GuzzleHttp\Client;
+use Exception;
+use GuzzleHttp\Exception\RequestException;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Log;
+use models\summit\Summit;
+/**
+ * Class MarketingAPI
+ * @package services\apis
+ */
+final class MarketingAPI implements IMarketingAPI
+{
+    /**
+     * @param string $api_url
+     * @param array $params
+     * @return mixed
+     * @throws Exception
+     */
+    protected function getEntity(string $api_url, array $params)
+    {
+        try {
+            $client = new Client();
```

Comment:
dont create the client here
please check implementation here
https://github.com/OpenStackweb/summit-api/blob/9f48390f7f3d33ab15963f3558722ce4049c9527/app/Services/Apis/MailApi.php#L56

Reply:
done